src/Sylius/Bundle/ResourceBundle/spec/Controller/RequestConfigurationFactorySpec.php

```diff
@@ -63,4 +63,50 @@ function it_creates_configuration_from_resource_metadata_and_request(
 
         $this->create($metadata, $request)->shouldHaveType(RequestConfiguration::class);
     }
+
+    function it_creates_configuration_without_default_settings(
+        MetadataInterface $metadata,
+        Request $request,
+        ParametersParser $parametersParser,
+        ParameterBag $headersBag,
+        ParameterBag $attributesBag
+    ) {
+        $this->getDefaultParameters()->shouldBeEqualTo([]);
+
+        $request->headers = $headersBag;
+        $request->attributes = $attributesBag;
+
+        $configuration = ['template' => ':Product:list.html.twig'];
+
+        $attributesBag->get('_sylius', [])->shouldBeCalled()->willReturn($configuration);
```

Comment:
No need to call ``shouldBeCalled()`` when calling ``willReturn(...)`` (exception: builders ;))